tests/containers/workbenches/rstudio/rstudio_test.py

```diff
@@ -153,3 +186,10 @@ def default(self, o):
 
 
 _ = StructuredMessage  # optional shortcut, to improve readability
+
+
+class TestCase():
+    def __init__(self, name, name_lc, value):
+        self.name = name
+        self.name_lc = name_lc
+        self.value = value
```

Comment:
I'd nest this under `def test_`, because this only applies for that one test and no other

and using `NamedTuple`, or `@dataclass` is better here, imo; avoids having to repeat the field names twice

* https://docs.python.org/3.12/tutorial/classes.html#odds-and-ends
* https://docs.python.org/3/library/typing.html#typing.NamedTuple

compared do my version, you dropped the `:str` type annotations ;p

Reply:
good point, updated